### PR TITLE
feature - add endpoint to get supplies of given multi assets

### PR DESCRIPTION
**Pull request by ozgrakkurt**
this PR adds /multiAsset/supply endpoint for getting supplies for given assets

### CODE REVIEW

**Comment by ozgrakkurt** [comment]
I'll also add the SQL query for the new version of db-sync before merging this

**Comment by pedromtcosta** [comment]
@ozgrakkurt looks good to me, but why is it a WIP? Also, please add the API docs to README

**Comment by ozgrakkurt** [comment]
@pedromtcosta Oh it's because I was going to add the query for the new db-sync version to this code as well, but in a comment. But now that I think about it, I can add that to the beta brach seperately so this is done, I am removing WIP. Also adding the API docs to README.

**Comment by pedromtcosta** [comment]
@ozgrakkurt what do you think of adding a "temporary" parameter on the query string to switch between the new and the old db-sync version? It could default to the old one, as we are currently using it for mainnet

**Comment by ozgrakkurt** [comment]
@pedromtcosta I think the best way to do this would be:
- merge this to main
- sync beta with main
- update it on the main
This way all branches have only the code they need. If this works for you, I can do this.

**Comment by pedromtcosta** [comment]
@ozgrakkurt makes sense, changes looks good to me then 👍

**Review by pedromtcosta — APPROVED** [review]

**Comment by ozgrakkurt** [comment]
can't merge this for some reason, I think linter is stuck. This is what I see in my browser:
image

**Comment by vsubhuman** [comment]
> can't merge this for some reason, I think linter is stuck. 

@ozgrakkurt , all good, it's waiting for an approval on latest changes

File: src/services/multiAssetSupply.ts
Comment (vsubhuman) [inline src/services/multiAssetSupply.ts:33]: @ozgrakkurt , this is for the current version of db-sync in production, correct?

Comment (ozgrakkurt) [reply src/services/multiAssetSupply.ts:33]: Yes, this is for the production/master version. Not the beta

**Review by vsubhuman — APPROVED** [review]

**Pull request merged**
